Makefile

```diff
@@ -9,6 +9,9 @@ build: Dockerfile
 docker-push: Dockerfile
 	docker push infoblox/helm:$(IMAGE_VERSION)
 
+version-major:
+	@echo $(shell echo ${IMAGE_VERSION} | cut -d '.' -f 1)
```

Comment:
It's not clear to me why this is wrapped. What does the extra shell invocation do?

Reply:
fixed